other/enforce-readwriteonce-pod/artifacthub-pkg.yml

```diff
@@ -28,4 +28,4 @@ readme: |
 annotations:
   kyverno/category: "Sample"
   kyverno/subject: "PersistentVolumeClaims"
-digest: 681fc0e98d14ae0a099580e221d9a467afa6d8e639b31cd1ac44e2275f7ebf83 
+digest: 2fb28419ac572f7a3d8ee50f6dd157b00b015638f6a9d06728783f35ffb6c7f0
```

Comment:
Why did you change the digest in this commit? The digest only gets changed when the policy is changed.

Reply:
actually the ci for linting was failing so I changed it